zipline/algorithm.py

```diff
@@ -2272,13 +2281,13 @@ def attach_pipeline(self, pipeline, name, chunks=None, eager=True):
     @api_method
     @require_initialized(PipelineOutputDuringInitialize())
     def pipeline_output(self, name):
-        """Get the results of the pipeline that was attached with the name:
-        ``name``.
+        """
+        Get results of the pipeline attached by with ``name``.  ``name``.
```

Comment:
`name` is duplicated at the end of this sentence. Maybe `Get results of the pipeline attached with name ``name``.`?